### PR TITLE
VT-2 add basic script

### PR DESCRIPTION
**Pull request by gbusila**

### CODE REVIEW

File: src/main/resources/sql/db_sql.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+drop table vesta_schema.user;
+drop table vesta_schema.user_role;
+drop table vesta_schema.role;
+
+create table vesta_schema.user(
+    id serial primary key,
+    name varchar (50) UNIQUE not null ,
```
Comment (ccaraian) [inline src/main/resources/sql/db_sql.sql]: what you mean by name? username or first name? if first name then why it need to be UNIQUE

File: src/main/resources/sql/db_sql.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    lastname varchar (50) unique  not null ,
```
Comment (ccaraian) [inline src/main/resources/sql/db_sql.sql]: lastname dosen't need to be unique and by naming convention if field is composed from two or more words they are separated by _ ex: lastname -> last_name

**Review by ccaraian — COMMENTED** [review]
Also by naming convention all sql keywords are written with uppercase
Ex: create table -> CREATE TABLE

**Review by ccaraian — APPROVED** [review]

**Review by vadimeladii — APPROVED** [review]
Good job @gbusila

**Pull request merged**